scripts.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# Reset
+_N_='\033[0m'     # Reset
+_R_='\033[0;31m'  # Red
+_G_='\033[0;32m'  # Green
+_Y_='\033[0;33m'  # Yellow
+_B_='\033[0;34m'  # Blue
+_P_='\033[0;35m'  # Purple
+_C_='\033[0;36m'  # Cyan
+_S_='\033[0;37m'  # Soft (Gray)
+
+DATABASE=./packages/database
+WEB=./apps/web
+[ $USER = "khang" ]   && SRC=~/dots/personal/.secrets/modtree
+[ $USER = "weiseng" ] && SRC=$REPOS/orbital/env
+
+# weiseng's config
+
+w_env() {
+  env_start
+	cp $SRC/.env.local $WEB/.env.local
+	cp $SRC/.env $DATABASE/.env
+  env_end
+}
+
+w_inv() {
+	cp $WEB/.env.local $SRC/.env.local
+	cp $DATABASE/.env $SRC/.env
+}
```

Comment:
want to add inv_start and inv_end here too

Reply:
done